### PR TITLE
Added movie route

### LINKED ISSUE #121: Add movies route
**Describe the feature**
Add movies route with JSON having `title` `genre` `rating` `imgUrl` `description` .

**Tasks Todo**
- [ ] Add  json file with appropiate data in `moviesData.json`.
- [ ] Add endpoint `/movies` in `movies.ts`.
- [ ] Add enpoint `/movies/:id` in `movies.ts`.

**Are you working on this feature? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by Samridhi-98**
# Related Issue

-   closes #121 

# Proposed Changes

- [x] Added  json file with appropiate data in `moviesData.json`.
- [x] Added endpoint `/movies` in `movies.ts`.
- [x] Added enpoint `/movies/:id` in `movies.ts`.

# Checklist

-   [x] Tests
-   [x] Translations

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@ShauryaAg  @paramsiddharth  Please review

File: data/moviedata.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "data": [{
+            "title": "3-idiots",
```
Comment (paramsiddharth) [inline data/moviedata.json:3]: Please fix the capitalization: `3 Idiots`.

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
The rest looks good to me.

**Comment by Samridhi-98** [comment]
@paramsiddharth done!

File: data/moviedata.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            "title": "Bajirao Mastani",
+            "director": " Sanjay Leela Bhansali",
+            "genre": "historical drama",
+            "rating": "7.9",
+            "description": "An account of the romance between the Maratha general, Baji Rao I and Mastani, princess of Bundelkhand."
+        },
+        {
+            "title": "Baahubali",
+            "director": " S.S. Rajamouli",
+            "genre": "action drama",
```
Comment (ShauryaAg) [inline data/moviedata.json:19]: `genre` should be a list. 

```js
"genre": ["action", "drama"]
```

**Review by ShauryaAg — CHANGES_REQUESTED** [review]

File: data/moviedata.json (hunk 2 above)
Comment (Samridhi-98) [reply data/moviedata.json]: Done!

File: data/moviedata.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{
+    "data": [{
```
Comment (ShauryaAg) [inline data/moviedata.json:2]: Put the `{` in the next line.

```json
"data": [
  {
  ...
  }
```

**Review by ShauryaAg — CHANGES_REQUESTED** [review]

File: data/moviedata.json (hunk 3 above)
Comment (Samridhi-98) [reply data/moviedata.json]: Done!

**Review by ShauryaAg — APPROVED** [review]

**Pull request merged**